bee-api/bee-rest-api/src/endpoints/auth.rs

```diff
@@ -0,0 +1,148 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::marker::PhantomData;
+
+use auth_helper::jwt::{Claims, JsonWebToken, TokenData};
+use axum::{
+    async_trait,
+    extract::{Extension, FromRequest, OriginalUri, RequestParts, TypedHeader},
+    headers::{authorization::Bearer, Authorization},
+    http::Uri,
+};
+use lazy_static::lazy_static;
+use regex::RegexSet;
+use serde::{Deserialize, Serialize};
+
+use crate::endpoints::{config::route_to_regex, error::ApiError, storage::StorageBackend, ApiArgsFullNode};
+
+pub const API_AUDIENCE_CLAIM: &str = "api";
+pub const DASHBOARD_AUDIENCE_CLAIM: &str = "dashboard";
```

Comment:
Should this be under the dashboard feature ?

Reply:
There was a reason to not add it (unfortunately I can't remember it), but after testing with + without the feature flag everything works fine. So I added the feature flag!